.circleci/config.yml.j2

```diff
@@ -39,7 +39,7 @@ jobs:
   platform-{{ version | replace(".", "-") }}:
     machine:
       image: ubuntu-1604:201903-01
-      resource_class: xlarge
+      resource_class: 2xlarge
```

Comment:
We ought to revert this if it has no effect on the race condition and tests passing.

Reply:
OK I will revert this after I try the houston reconfiguration and compare the results.